godot/Ingredient.gd

```diff
@@ -0,0 +1,11 @@
+class_name Ingredient
```

Comment:
I'd probably just make this more generic and call it `Item` but im okay with doing that in a later change :shrug:

Reply:
bleh I meant in the Inventory node, this is fine since its just the ingredients in general

Reply:
yeah i was debating whether to use item or ingredient im ok with either! ill just push this up for now :p